src/main/java/seedu/algobase/logic/commands/tag/DeleteTagCommand.java

```diff
@@ -45,6 +48,9 @@ public CommandResult execute(Model model, CommandHistory history) throws Command
         Tag tagToDelete = lastShownList.get(targetIndex.getZeroBased());
         model.deleteTags(tagToDelete);
         model.deleteTag(tagToDelete);
+        model.updateFilteredProblemList(PREDICATE_SHOW_ALL_PROBLEMS);
+        model.updateFilteredPlanList(Model.PREDICATE_SHOW_ALL_PLANS);
```

Comment:
Why are these two lines changed?

Reply:
Without these two lines the states of problems and plans will not refresh in the UI.